<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkVolumeVisualizationViewControls</class>
 <widget class="QWidget" name="QmitkVolumeVisualizationViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Volume visualization</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="volumeSelectionLabel">
       <property name="text">
        <string>Selected image:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QmitkSingleNodeSelectionWidget" name="volumeSelectionWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="enableRenderingCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Enable volume rendering of the selected image in the 3D view</string>
       </property>
       <property name="text">
        <string>Volume rendering</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblBlendMode">
       <property name="text">
        <string>Blend mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="blendMode">
       <property name="toolTip">
        <string>Switch between different blend modes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QmitkTransferFunctionGeneratorWidget" name="transferFunctionGeneratorWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkTransferFunctionWidget" name="transferFunctionWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSingleNodeSelectionWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkTransferFunctionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkTransferFunctionWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkTransferFunctionGeneratorWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkTransferFunctionGeneratorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
